### DISCUSSION TITLE
UniProt

### DISCUSSION THREAD

**Discussion by jwehr618** [KarchinLab/open-cravat#241] [Usability & Feature Requests]
Hello,

You would be able to pull from Unitprot the domain information in addition to what is already pulled in the OC output. What we would need is if there is a domain, the name of the domain, and the amino acid position. This is to determine whether the variant annotated falls in a domain or not. I work with the command line output, so would be great for it to be in the Excel output.

Thanks

**Comment by kmoad** [comment]
We are working on adding this.

**Comment by kmoad** [comment]
We have added a new annotator `uniprot_domain` that will return the domain a variant is found in. It is based on the domain bed file from https://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/genome_annotation_tracks/UP000005640_9606_beds/
